engines/dreamweb/titles.cpp

```diff
@@ -99,6 +101,42 @@ void DreamWebEngine::gettingShot() {
 }
 
 void DreamWebEngine::bibleQuote() {
+	const char *enStory = "And I heard a great voice out of the temple saying to the seven angels, "
+						"Go your ways and pour out the vails of the wrath of god upon the earth. "
+						"Book of revelation Chapter 16 verse 1.";
+	const char *frStory = "Puis j'entendis une voix forte qui venait du temple et disait aux sept anges: "
+						" «Allez verser sur la terre les sept coupes de la colère de Dieu.»"
+						"L'Apocalypse, chapitre 16, verset 1";
+	const char *esStory = "Oí una gran voz que decía desde el templo a los siete ángeles: "
+						"Id y derramad sobre la tierra las siete copas de la ira de Dios. "
+						"Libro del Apocalipsis, capítulo 16, versículo primero.";
+	const char *deStory = "Dann hörte ich, wie eine laute Stimme aus dem Tempel den sieben Engeln zurief: "
+						"Geht und gießt die sieben Schalen mit dem Zorn Gottes über die Erde. "
+						"Offenbarung des Johannes. Kapitel 16 Vers 1";
+	const char *itStory = "Udii una gran voce dal tempio che diceva ai sette angeli: "
+						"Andate e versate sulla terra le sette coppe dell’ira di Dio. "
+						"libro della rivelazione. capitolo 16 versetto 1";
+
+	Common::TextToSpeechManager *ttsMan = g_system->getTextToSpeechManager();
+	const char *theStory;
+	switch(getLanguage()) {
+	case Common::ES_ESP:
+		theStory = esStory;
+		break;
+	case Common::FR_FRA:
+		theStory = frStory;
+		break;
+	case Common::IT_ITA:
+		theStory = itStory;
+		break;
+	case Common::DE_DEU:
+		theStory = deStory;
+		break;
+	default:
+		theStory = enStory;
+		break;
+	}
+
```

Comment:
Apparently there are Czech and Russian fan translations as well (we can see those in the detection table).
If we want to default to English for those here, we would need to temporarily change the language of the TextToSpeechManager to English.